fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java

```diff
@@ -104,7 +104,7 @@ public ActionAuthorizationInfo checkWithCookie(HttpServletRequest request,
     protected void addSession(HttpServletRequest request, HttpServletResponse response, SessionValue value) {
         String key = UUID.randomUUID().toString();
         Cookie cookie = new Cookie(PALO_SESSION_ID, key);
-        cookie.setSecure(false);
+        cookie.setSecure(true);
```

Comment:
`cookie.setSecure(true)` means Cookies will only be passed under the HTTPS protocol, which will make doris UI login can not work.

Reply:
Maybe  we can set the cookie secure flag in the fe.config ,by default the values is false.

Reply:
we can use enale_https to set cookie.setSecure true or false

Reply:
![image](https://github.com/apache/doris/assets/71873108/5144cdc8-13ea-4c32-9e65-1bddecbc49f1)
test case success